docker/rbac.providers.ldap.Dockerfile

```diff
@@ -47,8 +47,9 @@ WORKDIR /project/hyperledger-rbac
 # Container-specific dependencies are installed separately for
 # optimizing caching
 RUN pip3 install \
-        ldap3==2.5.1 \
+        rethinkdb \
+        ldap3 \
         pyasn1==0.4.4 \
-        pytz==2018.6
+        pytz
```

Comment:
Why remove the versions from some dependencies?

Reply:
oops, my bad, reverting.